src/config.js

```diff
@@ -6,3 +10,40 @@ export const HOSTNAME = process.env.HOSTNAME;
 export const PHOTON_SERVICE_NAME = process.env.PHOTON_SERVICE_NAME;
 export const NOMINATIM_SERVICE_NAME = process.env.NOMINATIM_SERVICE_NAME;
 export const FILE_SERVICE_NAME = process.env.FILE_SERVICE_NAME;
+
+// Load .env files in a similar way to the bikehopper-ui frontend
+// .env.development.local, .env.development, .env, etc
+//
+// We should probably only use this for development and rely on Docker
+// supplying the env vars in production
+
+const NODE_ENV = process.env.NODE_ENV;
+if (!NODE_ENV) {
+  throw new Error(
+    'The NODE_ENV environment variable is required but was not specified.'
+  );
+}
+
+const dotenvPath = '.env';
```

Comment:
this all seems needless complex. i've never done any of this when using dotenv. why not just do the standard thing:
```javascript
import * as dotenv from 'dotenv';
dotenv.config();
```